Civi/RemoteParticipant/Event/GetCancelParticipantFormEvent.php

```diff
@@ -30,10 +30,10 @@ class GetCancelParticipantFormEvent extends GetParticipantFormEventBase
     /**
      * Get the token usage key for this event type
      *
-     * @return string
+     * @return array
      */
-    protected function getTokenUsage()
+    protected function getTokenUsages()
```

Comment:
We could have a legacy ``getTokenUsage`` function lying around, implemented as something along the lines of 
```
return reset($this->getTokenUsages())
```
Just in case somebody was using it already extending ``GetCancelParticipantFormEvent``... And ideally we'd add a warning if it's more than one element. 

But I couldn't find any such implementation in our code base, so it's not too important.